common-lib/src/cmr/common/util.clj

```diff
@@ -616,10 +616,14 @@
 
 (defn map-matches-path-values?
   "Returns true if the map matches the given path values. Path values are
-  described in the map->path-values function documentation."
+  described in the map->path-values function documentation.  When a value is sequential,
+  any map which matches at least one of the contained values will be considered to match
+  (i.e. the values are ORed)"
   [path-values m]
   (every? (fn [[path value]]
-            (= (get-in m path) value))
+            (or
+              (when (sequential? value) (some #(= (get-in m path) %) value))
```

Comment:
Regarding the unit test comment: in particular, a test for `map-matches-path-values?` that checks for both maps and sequences of maps (and whatever else might get passed to `map-matches-path-values?` ...)